CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## [0.18.4](https://github.com/eclipse/lemminx/milestone/26?closed=1) (February 14, 2022)
+
+### Enhancements
+
+- Eliminate need for downloading just-released binary checksums.. See [#650](https://github.com/redhat-developer/vscode-xml/pull/650).
```

Comment:
I would remove the extra "." at the end of "checksums" and place this entry under a "Build" heading.

Reply:
fixed